README.md

```diff
@@ -41,59 +41,83 @@ These [values are what we value](http://www.slideshare.net/reed2001/culture-1798
-
-### Communicate
-We hire team members who we believe will make us smarter and better at what we're trying to do. In order to succeed at becoming smarter and better as a whole, it's essential for us to communicate.
+In practice, this is challenging. In order to go get results, we must understand current goals and priorities, and cost/benefit tradeoffs (ie. go for the quick win that costs little, or spend more time on a robust solution?). Our core value of "Default to Transparency" should ensure that the information you need is available to you, but if it isn't, go seek it out.
 
-What does communication mean at Bonusly?
+### Be an excellent teammate
+* Be proactive and assertive
+* Solicit feedback and input
+* Be [radically candid](https://www.radicalcandor.com/about-radical-candor/)
+* Listen first
+* Assume the best
+* Be there for others
+* Teach & encourage
+* Solve problems together
 
-* You actively solicit input on work you’re doing.
-* You listen before giving your opinion.
-* You contribute your ideas.
-* You keep your cool in stressful interactions.
-* You treat your colleagues with respect at all times, especially times of disagreement.
-* You have the courage to contribute on topics, even if they are not in your realm of responsibilities or expertise.
+We believe that success is achieved by people working on teams. As such, it is critical that we be the best teammates possible to one another. This means being proactive and assertive in your communication: asking for what you need, providing candid, constructive feedback, and voicing your opinions and ideas. It also means listening to, supporting, and encouraging your colleagues -- especially at times of disagreement and stress.
 
 ### Default to transparency
+* Take responsibility to share information that others need (with context)
+* Share what you’ve learned
+* Be vulnerable - talk about mistakes, failures, and bad news
+* Create an environment where it’s easy to get information
+
 (We took this phrasing from [Buffer](https://open.bufferapp.com/buffer-values/). Thanks, Buffer.)
 
 We don’t just occasionally share our internal metrics -- we make them available in real time. Any member of Bonusly can log in to our internal dashboard and see our up-to-date metrics: revenue, burn, churn, runway, etc. This ensures that people have the information they need to come up with novel solutions and act on them.
 
+Just posting information isn't enough, however. Sharing _context_ is also necessary. So, we have monthly all-hands meetings and post explainers in Slack so that team members don't just see our numbers, priorities, and goals, but understand what they mean.
+
 On an individual level, we candidly share stories of our successes, failures, challenges, and interests so that we can learn from and support one another.
 
```

Comment:
Somewhere in this description address the fact that "default" doesn't mean passive?

Address whether this is a value that we put into practice externally as well and/or in the product for our customers/users?

Add in a sentence or two about why we think this value is important? I really liked that in the value above.

I think people mentioned a lot that they strongly associated this value with the business number sharing, so it might be helpful to expand on the individual level with more detailed explanations of what this might be. I'm thinking "sharing the results of an experiment" type of level of detail as one example.